bentoml/_internal/server/schedule_strategy.py

```diff
@@ -0,0 +1,16 @@
+import logging
+from timeit import default_timer
+from typing import TYPE_CHECKING
+from contextvars import ContextVar
+
+if TYPE_CHECKING:
+    from .. import external_typing as ext
+
+import abc
+
+
+class SchedulingStrategy(abc.ABC):
```

Comment:
What's this for?

Reply:
The is the base class for extending custom scheduling strategy